src/templates/base/base_chapter.html

```diff
@@ -65,14 +65,7 @@ <h2 id="explore-results">
       {{ self.explore_the_results() }}
     </a>
   </h2>
-  <a class="alt btn" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
-    <svg width="18" height="18" role="img">
-      <title>{{ self.discuss_this_chapter() }}</title>
-      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#comment"></use>
-    </svg>
-    <span id="num_comments">{{ self.comments_fallback() }}</span> <span data-translation id="comment-singular">{{ self.comment() }}</span>
-    <span data-translation id="comment-plural">{{ self.comments() }}</span>
-  </a>
```

Comment:
I still wonder if we should leave this in place here (and add the other new ones as well) and then have the floating CTAs stop before this (or could have it vanish or merge into this but that'll be more tricky).

I've a demo of what I'm talking about here: https://20210517t231435-dot-webalmanac.uk.r.appspot.com/en/2020/fonts

I've also moved the CTAs to be horizontal as feel they cover up less space in the chapter this way and so are less irritating.

What do you think? @rviscomi any comments since you originally asked for this?

I can commit my changes to this branch if we agree this is the way forward as probably easier than explaining all the small changes I did to make this work.